src/reducers/rootReducer.js

```diff
@@ -1,30 +1,46 @@
 import { combineReducers } from "redux";
-import { SET_PAGE, SET_EVENT } from '../constants/Constants';
+import { SET_EVENT, SET_EVENTS, SET_USER, LOGOUT } from '../constants/Constants';
```

Comment:
How do `SET_EVENT(S)` replace `SET_PAGE` and `SET_EVENT`

Reply:
It doesn't replace it, it's something else.

Previously the redux store kept the "current page". Now that we are using react router we dont need set page

Set event is still there.

And set events is to set the list of event objects